api/types/src/transaction.rs

```diff
@@ -1061,6 +1063,24 @@ impl VerifyInput for MultisigPayload {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
+pub struct AutomationTransactionPayload {
+    pub inner_payload: EntryFunctionPayload,
```

Comment:
At move layer this would be `Vec<u8>`, do we have appropriate conversion when making a call to `register` method in Move layer?

Reply:
yes it is correctly converted.